src/tracing/perfetto/BUILD.gn

```diff
@@ -75,7 +75,6 @@ static_library("perfetto") {
   public_deps = [
     ":perfetto_tracing",
     "${chip_root}/src/lib/address_resolve",
-    "${chip_root}/src/tracing",
     "${chip_root}/src/transport",
```

Comment:
why was this removed? I see `src/tracing/perfetto/perfetto_tracing.h` having an include of `#include <tracing/backend.h>` so I assume the dependencies are needed.